### ISSUE TITLE
Proton 5.13 on Steam Flatpak doesn't run games

### ISSUE THREAD

**Issue by odesb** [ValveSoftware/Proton#4268]
Games tested:

- Age of Empires 2: Definitive Edition
- Resident Evil 7
- Prison Architect
- Dota 2

All games crash at boot; with Proton 5.0-9 I can run them. My distribution of Steam is the Flatpak one.

[Age of Empires 2: DE flatpak log](https://gist.github.com/xisbroken/5db33cf164144137978db93bc9755942)
[System Information](https://gist.github.com/xisbroken/fa157e82398c2151155fc2a51a27db18)

**Comment by John-Gee** [comment]
Is it because 5.13 uses Pressure Vessel? Maybe a container inside a container is no-go.

**Comment by Sporif** [comment]
Pressure Vessel is blocked by Flatpak's seccomp policy: https://github.com/flatpak/flatpak/blob/7d30de344c2065d95aa444b8a59ae0fb44a06929/common/flatpak-run.c#L2681

**Comment by jomarocas** [comment]
is aduplicated and is not related by flatpak, in general no working

**Comment by odesb** [comment]
I confirm it is caused by flatpak. I uninstalled from Flatpak and installed the distro-supplied package (Fedora 32). Works now!

**Comment by annulen** [comment]
>Is it because 5.13 uses Pressure Vessel? Maybe a container inside a container is no-go.

Is there any information on what is that "Pressure Vessel" and what system requirements does it have? Or is it possible to disable it? I'm using Steam on Gentoo without any flatpacks and Proton 5.13 fails to start anything.

**Comment by John-Gee** [comment]
You can find this link in the release notes:

https://steamcommunity.com/app/221410/discussions/0/1638675549018366706/

Your issue may be that Pressure Vessel blocks not only containers like flatpak, but also Vulkan overlays like vkbastalt and mangohud. I believe there is some workaround to disable Pressure Vessel, but it'd be best for Valve to provide an option (or not block stuff).

**Comment by kisak-valve** [comment]
Closing in favor of #4283. Please continue the discussion on that issue report.

**Comment by nickavem** [comment]
Sorry about reopening an old issue, but I can't hyperlink or find #4283. 

I love the Steam flatpak (and maybe even hope valve upstreams it at some point 😉) and I really hope this issue gets fixed soon. Community Proton Builds built on the actual flatpak runtime seem to bypass a lot of these issues. This can easily be installed with `flatpak install com.valvesoftware.Steam.CompatibilityTool.Proton` and should show up with all the other proton versions in *Force the use of a specific compatibility tool*.

**Comment by kisak-valve** [comment]
Hello @nickavem, https://github.com/ValveSoftware/Proton/issues/4283 has moved to https://github.com/ValveSoftware/steam-runtime/issues/294 because this is an issue with Steam Linux Runtime - Soldier.

**Comment by nickavem** [comment]
Ah, thank you for clarifying. Think I should cc this over there?

Sent from my desktop.
Andrew "Andy" James Thurman
980-333-5159
